examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2541,9 +2541,9 @@ pub extern "C" fn channel_loser(_handle: u64) {
 #[no_mangle]
 pub extern "C" fn http_oak_main(http_channel: u64) {
```

Comment:
Could this whole thing be replaced with `entrypoint_command_handler!(http_oak_main => StaticHttpHandler)`?

Reply:
Yes but I was trying to keep the abitests using the low-level ABI rather than the helper methods. I am actually going to create a separate PR about lowlevel and high-level APIs to clarify the distinction.

Reply:
I see. Thanks. I was just curious.